include/livox_laser_simulation/csv_reader.hpp

```diff
@@ -0,0 +1,48 @@
+//
+// Created by lfc on 2021/3/1.
+//
+
+#ifndef SRC_GAZEBO_CSV_READER_HPP
+#define SRC_GAZEBO_CSV_READER_HPP
+
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <vector>
+
+class CsvReader {
```

Comment:
Maybe add a comment that this is designed to reading a comma separated list of numeric values from file, as opposed to reading a generic CSV file.

Reply:
Does [b898d54](https://github.com/Airtonomy/flight-px4-gazebo/pull/3/commits/b898d54dee789e3e89028c1105df9c3deb808e3f) work?

Reply:
:+1: